pepper-apis/src/main/java/org/broadinstitute/ddp/model/dsm/ParticipantStatusTrackingInfo.java

```diff
@@ -103,6 +107,7 @@ public ParticipantStatusTrackingInfo(
         }
 
         this.kits = kits;
+        this.workflows = workflows;
```

Comment:
This is not a safe operation, and could lead to undefined behavior. This should be a copy instead of reference assignment.